packages/tasit-action/src/Contract.js

```diff
@@ -75,6 +75,16 @@ export class Contract extends Subscription {
         this.#ethersContract,
         this
       );
+
+    //if (!this.#subscription.subscribedEventNames().includes("error")) {
+    const errorListener = message => {
+      const { error } = message;
+      this._emitErrorEvent(new Error(`${error.message}`));
+    };
+
+    this.#subscription.on("error", errorListener);
```

Comment:
Does the subscription mind if the contract claims its only error listener? We should make a test for what happens if the developer then sets a different error listener explicitly on the subscription after first setting it on the contract

Reply:
Great! Could we extract this thread to: https://github.com/tasitlabs/TasitSDK/issues/164 ?

Reply:
Yep!